src/raft.rs

```diff
@@ -418,9 +439,6 @@ impl<T: Storage> Raft<T> {
         for (_, pr) in self.mut_prs().iter_mut() {
             pr.commit_group_id = 0;
         }
-        if StateRole::Leader == self.state && self.maybe_commit() {
```

Comment:
Why remove it?

Reply:
If using group commit, then clearing group id will make commit index become smallest one, so no need to commit; if not using group commit, clearing group id has no effect.